### PR TITLE
Update anndata

### PR DESCRIPTION
**Pull request by ivelet**
FOR CONTRIBUTOR:
* [ ] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/anndata/inspect.xml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<tool id="anndata_inspect" name="Inspect AnnData" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@">
+<tool id="anndata_inspect" name="Inspect AnnData" version="@VERSION@+@GALAXY_VERSION@">
```
Comment (bgruening) [inline tools/anndata/inspect.xml:1]: the previous version looked correct

**Comment by pavanvidem** [comment]
@bgruening  all tests failed with the following error. What does it mean?
```
Job in error state.. tool_id: anndata_export, exit_code: 125, stderr: docker: invalid spec: ::rw: empty section between colons.
See 'docker run --help'.

WARNING:galaxy.datatypes.binary:<galaxy.datatypes.binary.Loom object at 0x7fe688b15590>, set_meta Exception: Unable to open file (file signature not found)
```

**Comment by bernt-matthias** [comment]
Restarted the tests.

**Comment by pavanvidem** [comment]
please close this. These changes were included in another PR https://github.com/galaxyproject/tools-iuc/pull/5633

**Pull request closed without merging**